stingray/utils.py

```diff
@@ -13,6 +15,11 @@ def jit(fun):
         return fun
 
 
+class UnrecognizedMethod(Exception):
+    pass
+
+
+
```

Comment:
Having a custom exception class in utils sounds like a good idea to me. However, I did not understand its use case in this case.